src/core/bot.ts

```diff
@@ -121,3 +126,7 @@ export class YveBot {
 YveBot.prototype.types = new Types;
 YveBot.prototype.actions = new Actions;
 YveBot.prototype.validators = new Validators;
+
+
+YveBot.Exceptions = Exceptions;
+YveBot.WaitForUserInputExecutor = WaitForUserInput;
```

Comment:
`YveBot.executors = new Executors`
to be accessed by `YveBot.executors.WaitForUserInput`